resources/js/components/modules/moduleContent.vue

```diff
@@ -1,11 +1,15 @@
 <template>
-    <div class="row">
-        <div class="module-content"></div>
-    </div>
+  <div class="module-content">
+    <CardFlip></CardFlip>
+  </div>
 </template>
 
 <script>
+import CardFlip from './media/CardFlip'
```

Comment:
to follow convention this should be `cardFlop`

Reply:
I thought our convention for vue was capital first name of each word.

Reply:
for views not for components